cfn-resources/ldap-configuration/docs/README.md

```diff
@@ -52,7 +52,7 @@ Properties:
 
 #### Profile
 
-The profile is defined in AWS Secret manager. See [Secret Manager Profile setup](../../../examples/profile-secret.yaml).
+Profile used to provide credentials information, (a secret with the cfn/atlas/profile/{Profile}, is required), if not provided default is used
```

Comment:
For reference https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/724#discussion_r1338264116, don't see the need of adjusting but to keep in mind if case appears again.